### ISSUE TITLE
MinIO Object Storage error when loading images

### ISSUE THREAD

**Issue by aqua-pro** [LemmyNet/lemmy-ui#2292]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

I have a MinIO S3 Object Storage Server and wanted to start using it for my Lemmy instance (wip) since the start. Everything works normally for Lemmy. When I enable S3 Object Storage and upload images (site icon, site banner) everything works as expected too, images are uploaded correctly, I see them in the bucket in MinIO, and the images are even publicly accessible. However, after some time, cache, and/or restarting Lemmy, the web UI will be trying to actually download those images to serve to the user, and it will return an error.

### Steps to Reproduce

1. Setup Lemmy
2. Setup MinIO and bucket
3. Connect Lemmy to MinIO
4. Upload Site Banner or Site Icon
5. Restart Lemmy
6. Control + F5 to force refresh the Lemmy page

### Technical Details

This is the error returned

[Error: Input buffer contains unsupported image format]

### Lemmy Instance Version

0.18.5

**Comment by aqua-pro** [comment]
I've further discovered that this happens with or without S3, and it happens only after restarting Lemmy

**Comment by madeofstown** [comment]
I'm having the server error issue after uploading a site image and then restarting the server. version 0.19.0

**Comment by Tmpod** [comment]
Ideally the UI shouldn't completely die when images aren't reachable or are returned invalid. Mlmym (also SSR'd) works well in this scenario. Each time there's some issue with pict-rs or the S3 provider (Scaleway, I see you...), or if I'm just doing something to pict-rs, I always have my main web interface going down, while everything else still works. A bit frustrating.

I might take a stab at this, if I can find some time.

**Comment by Nothing4You** [comment]
This issue appears to be a duplicate of #2211.

**Comment by Tmpod** [comment]
True, my bad!
